Meta/new-project.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+script_path=$(cd -P -- "$(dirname -- "$0")" && pwd -P)
+cd "${script_path}/.." || exit 1
```

Comment:
`set -e` already kills the script when a command fails, this `|| exit 1` is redundant.